abstract_specification.md

```diff
@@ -618,6 +618,10 @@ _Note:_ this API was preceded by another (un-versioned) API draft, which is coll
 ## Indexing and slicing
 
 * In the above `read()` methods, indexing by an empty list of IDs must result in zero-length query results.
+* Negative indices are not to be interpeted as aliases for positive indices (as is common in Python) or as exclusionary (as is common in R).
+* Slices are doubly inclusive -- e.g. slicing with bounds 2 and 4 includes array indices 2, 3, and 4.
```

Comment:
Could you also use the standard mathematical term for these - "closed ranges" - in addition to the colloquial "doubly inclusive". It is much more commonly used in my experience.

Ie., "Slices define a closed range, ie., are doubly inclusive of specified values"